### PR TITLE
Add convenience functions for planets + moons. Add build_time function.

### PR DESCRIPTION
**Pull request by CptBaradon**
Added some convenience functions:
- name_by_planet_id
- id_by_planet_coords
- id_by_moon_name
- id_by_moon_coords

Added calculation of build time in seconds based on resources.

### CODE REVIEW

**Review by PiecePaperCode — APPROVED** [review]
love it great job

**Comment by CptBaradon** [comment]
I added a new commit to this PR and merged the current dev aswell.
Should prevent merge conflicts.

**Comment by PiecePaperCode** [comment]
Now we need to update the readme aswell

File: ogame/__init__.py [hunk 1]
```diff
@@ -249,6 +274,18 @@ class resources:
 
         return resources
 
+    def build_time(self, id, resources):
```
Comment (PiecePaperCode) [inline ogame/__init__.py:277]: This is a function bedder suited on an different File named convenience or shortcut like in django.
All the convenience would be there that a bot dev would use all the time. __init__ is more for request

**Review by PiecePaperCode — CHANGES_REQUESTED** [review]

File: ogame/__init__.py (hunk 1 above)
Comment (PiecePaperCode) [reply ogame/__init__.py:277]: price from constants would be moved there too

**Comment by PiecePaperCode** [comment]
closed due to inactivety

**Pull request closed without merging**